server/routes/ticket.py

```
@@ -302,5 +300,333 @@ def post(self):
+                    )
+
+            db.session.commit()
+
+            return restful_response(
+                status="success",
+                message="Ticket updated successfully",
+                data=ticket.to_dict(),
+                status_code=200,
+            )
+
+        except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)